src/routes/board/components/stats/Stats.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import classes from "./Stats.module.css";
+
+const Stats = ({ money, bid, strength, pot }) => (
+  <div className={classes.container}>
+    {money > -1 && <h2 className={classes.text}>Money: {money}</h2>}
+    {bid > -1 && <h2 className={classes.text}>Bid: {bid} </h2>}
+    {strength && <h2 className={classes.text}>{strength} </h2>}
+    {pot > -1 && <h2 className={classes.text}>Pot: {pot} </h2>}
```

Comment:
What this minus one means. I would clarify it in a larger project in a hoc. It may not make sense now but it's a good practice.

Reply:
HOC added and appropriate prop used.